jobs/ci-kubernetes-e2e-gce-federation.env

```diff
@@ -8,7 +8,7 @@ KUBE_GCS_RELEASE_BUCKET=kubernetes-federation-release
 KUBE_GCS_DEV_RELEASE_BUCKET=kubernetes-federation-release
 KUBE_NODE_OS_DISTRIBUTION=debian
 
-GINKGO_TEST_ARGS=--ginkgo.focus=\[Feature:Federation\]
+GINKGO_TEST_ARGS=--ginkgo.focus=\[Feature:Federation\] --ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]
```

Comment:
This looks good to me.

But I am not sure if we want to split the CRI and GCI tests, and all other tests in the future. Regular GCE tests are a good indicators of failures. So we want them to run as fast as possible. It is probably not necessary to split the other tests. They can take a while.